pypit/arsciexp.py

```diff
@@ -404,8 +406,9 @@ def MasterFlatField(self, fitsdict, det):
                         arproc.slit_profile(self, self.GetMasterFrame("pixelflat", det),
                                             det, ntcky=settings.argflag['reduce']['flatfield']['params'][0])
                     # If some slit profiles/blaze functions need to be extrapolated, do that now
-                    if np.sum(extrap_slit) != 0.0:
-                        slit_profiles, mstracenrm, msblaze = arproc.slit_profile_pca(self, self.GetMasterFrame("pixelflat", det), det, msblaze, extrap_slit, slit_profiles)
+                    if False:  # This is not a good idea for multislit, JXP thinks..
```

Comment:
This is nit-picky, but why use the `if False` syntax?  Why not just comment it out?

Reply:
For me, I suspect a commented out code may never come
back (i.e. it is more likely to simply be deleted).  Whereas
an ```if False``` might return at a moment's notice.  :)

Reply:
I feel a little uncomfortable leaving this as is, since I may forget to take it out for ARMED. So, in the meantime, I would suggest we replace `if False` with `if ARMED` (also, with line 470 below).

Reply:
done